### ISSUE TITLE
[crates] Default value of None to sleep_time when using CratesCmd

### ISSUE THREAD

**Issue by acs** [grimoirelab/perceval#265]
```
(acs@dellx) ~ $ perceval -g crates
....

[2017-12-22 18:43:56,167 - perceval.backends.mozilla.crates - WARNING] - Connection was lost, the backend will sleep for Nones before starting again
```

and it never restarts.

The workaround now is to pass a value for the param:

`(acs@dellx) ~ $ perceval -g crates --sleep-time 10
`

**Comment by sduenas** [comment]
That backend is not part of this repository. Open it in `perceval-mozilla`.

**Comment by acs** [comment]
https://github.com/grimoirelab/perceval-mozilla/issues/13
